team/jesslyn-difiori

```diff
@@ -0,0 +1,22 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Jesslyn Di Fiori
+  avatar: https://avatars.githubusercontent.com/jkdifiori
+  role: Technical Writer
+  organizations:
+    - name: National Snow and Ice Data Center (NSIDC)
+  bio: >
+    Jesslyn is a Technical Writer at the National Snow and Ice Data Center, where she writes and maintains scientific documentation, metadata, and web content for cryospheric data. She works primarily with NASA SnowEx, MODIS, and VIIRS data sets. 
+  expertise:     
+- Scientific Communication
+    - Data Visualization
```

Comment:
```suggestion
  expertise:     
    - Scientific Communication
    - Data Visualization
```

be sure to match indentation levels

Reply:
(if you click the 'commit suggestion' button above should be all set. the formatting is very picky!)